DESCRIPTION

```diff
@@ -11,9 +11,18 @@ Authors@R:
            role = c("aut", "cre"),
            email = "mwgerber@gmail.com")
 Description: What the package does (one paragraph).
-License: GPL-3
+License: GPL-3 + file LICENSE
```

Comment:
Since we have a license file we should point to it. From R package book:
```
The License field can be either a standard abbreviation for an open source license, like GPL-2 or BSD, or a pointer to a file containing more information, file LICENSE. The license is really only important if you’re planning on releasing your package. If you don’t, you can ignore this section. If you want to make it clear that your package is not open source, use License: file LICENSE and then create a file called LICENSE, containing for example:
```